itest/lnd_multi-hop_force_close_test.go

```diff
@@ -422,8 +422,7 @@ func runLocalClaimOutgoingHTLC(ht *lntest.HarnessTest,
 	if ht.IsNeutrinoBackend() {
 		numSweeps = 2
 	}
-
-	flakeSkipPendingSweepsCheckDarwin(ht, bob, numSweeps)
+	ht.AssertNumPendingSweeps(bob, numSweeps)
```

Comment:
Prior to the `LockIn` check, here Bob would have an HTLC on his pending remote commit if the test runs quickly, causing him to create another request that sweeps his anchor output from the pending remote commit, which fails the test.